exp/views/study.py

```diff
@@ -972,14 +979,16 @@ def user_can_view_previews(self):
         except Study.DoesNotExist:
             return False
 
+        print(child.user, request.user)
```

Comment:
Just checking that you meant to leave this in. Or was it just for testing?

Reply:
Whoops.  I'll put in a change for that.